src/app/Main/Body/QueryPage/Results/Charts/BarChartResults.component.tsx

```diff
@@ -7,6 +7,8 @@ import { ApexChartData } from "./ApexChart.model";
 export function BarChartResultsComponent(props: {
     chartData: ApexChartData;
     getPngDownloadUriRef: MutableRefObject<(() => Promise<string>) | null>;
+    lableXaxis: string;

```

Comment:
better name would be xAxisLabel, yAxisLabel. In general the more generic part comes first and specific part last. E.g. data => dataTable => dataTableHeader => dataTableHeaderLabel => dataTableHeaderLabelSize and so on.

Reply:
Done.